cg/models/pipeline_qc_metrics/pipeline_qc_metrics.py

```diff
@@ -0,0 +1,111 @@
+from pydantic import BaseModel
+
+
+class LaneMetrics(BaseModel):
+    """Model for lane specific sequencing metrics."""
+
+    lane: int
+    number_of_reads: int
+    _yield: int | None = None
+
+
+class ConcatenatedLaneMetrics(BaseModel):
+    """Model for concatenated lane sequencing metrics."""
+
+    number_of_reads: int
+    _yield: int | None = None
+
+
+class DNACoverageMetrics(BaseModel):
```

Comment:
```suggestion
class DNACoverageMetric(BaseModel):
```
Reserve plural for lists